src/Tizen.NUI/src/public/AccessibilityManager.cs

```diff
@@ -31,11 +41,20 @@ internal AccessibilityManager(global::System.IntPtr cPtr, bool cMemoryOwn) : bas
             return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.swigCPtr;
         }
 
+        /// This will not be public opened.
+        [EditorBrowsable(EditorBrowsableState.Never)]
         protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
         {
             Interop.AccessibilityManage.delete_AccessibilityManager(swigCPtr);
         }
 
+        internal static AccessibilityManager GetAccessibilityManagerFromPtr(global::System.IntPtr cPtr)
```

Comment:
Please check if this API is really used. because this is "internal" so user can not use this. if this is not used in NUI. please remove it.

Reply:
Okay, removed  it.
It was used in the signal internally of the previous patch, but now it's unnecessary. So, I removed it in the next patch.